library/src/main/java/org/xmtp/android/library/Contacts.kt

```diff
@@ -47,15 +47,15 @@ enum class EntryType {
         fun toFfiConsentEntityType(option: EntryType): FfiConsentEntityType {
             return when (option) {
                 EntryType.ADDRESS -> FfiConsentEntityType.ADDRESS
-                EntryType.GROUP_ID -> FfiConsentEntityType.GROUP_ID
+                EntryType.GROUP_ID -> FfiConsentEntityType.CONVERSATION_ID
```

Comment:
I'm going to do the breaking change to rename these to `conversation_id` in a follow up.